gulpfile.js

```diff
@@ -14,14 +15,19 @@ var gulp = require('gulp'),
   replace = require('gulp-string-replace'),
   sourcemaps = require('gulp-sourcemaps'),
   cp = require('child_process'),
-  exec = require('gulp-exec'),
+  del = require('del'),
+  exec = require('gulp-exec').exec,
+  KarmaServer = require('karma').Server,
+  lessCompiler = require('gulp-less'),
+  stylelint = require ('gulp-stylelint'),
+  cssmin = require ('gulp-cssmin'),
   ngc = require('gulp-ngc'),
-  changed = require('gulp-changed'),
```

Comment:
@mindreeper2420 The gulp build is failing with `changed` not found.  `gulp-changed` is used at https://github.com/mindreeper2420/fabric8-planner/blob/1f4abc328cc4ec741d065d609cc0844ca43f74ab/gulpfile.js#L52

Reply:
fixed - reference re-added.